package.json

```diff
@@ -1,27 +1,40 @@
 {
-  "name": "task1",
+  "name": "single_page_application",
   "version": "1.0.0",
-  "description": "Task 1 in React mentoring",
+  "description": "React mentoring",
   "main": "index.js",
   "scripts": {
-    "start server": "node server/server.js",
-    "build": "webpack --mode production"
+    "start": "webpack-dev-server --open --hot --config webpack.dev.js",
+    "build": "webpack -p --config webpack.prod.js",
+    "test": "jest"
   },
   "author": "Artur Liup",
   "license": "MIT",
   "dependencies": {
+    "clean-webpack-plugin": "^3.0.0",
+    "express": "latest",
+    "optimize-css-assets-webpack-plugin": "^5.0.3",
     "react": "^16.11.0",
     "react-dom": "^16.11.0",
-    "express": "latest"
+    "react-redux": "^7.1.1",
+    "react-router": "^5.1.2",
+    "redux": "^4.0.4",
+    "source-map-explorer": "^2.1.0"
   },
   "devDependencies": {
     "babel-core": "^6.26.3",
     "babel-loader": "^7.0.0",
     "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.24.1",
     "html-webpack-plugin": "^3.2.0",
+    "jest": "^24.9.0",
+    "mini-css-extract-plugin": "^0.8.0",
+    "node-env-run": "^3.0.2",
+    "redux-devtools": "^3.5.0",
```

Comment:
this tool you will call in the browser, so let's move it to dependencies.

Reply:
done